Antidote/ContactCellView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  ContactCellView.swift
+//  Antidote
+//
+//  Created by Yoba on 04/12/2016.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+import SnapKit
+
+class ContactCellView: NSTableCellView {
+    var userAvatar: NSImageView!
+    var usernameTextField: NSTextField!
+    var lastMessageTextField: NSTextField!
+    var callButton: NSButton!
+    var optionButton: NSButton!
+    
+    override init(frame: NSRect) {
+        super.init(frame: frame)
+        
+        self.createSubviews()
+        //self.setupConstraints()
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+    
+    override func viewDidMoveToSuperview() {
+        setupConstraints()
+    }
+}
+
+fileprivate extension ContactCellView {
+    func createSubviews() {
+        userAvatar = NSImageView()
+        addSubview(userAvatar)
+        
+        usernameTextField = LabelView()
+        usernameTextField.font = NSFont(name: "System Font", size: 20)
+        addSubview(usernameTextField)
+        
+        lastMessageTextField = LabelView()
+        lastMessageTextField.textColor = NSColor.gray
+        addSubview(lastMessageTextField)
+        
+        callButton = NSButton()
+        callButton.title = "Call"
```

Comment:
It would be nice to make app localizable from the start. For that you have to:

1. Create `Localizable.strings` file (you can choose "strings" file template in Xcode when creating a file).
2. In `Localizable.strings` file add strings with following format:
```swift
/* Button on contacts screen used to make a call */
"contacts_call_button" = "Call";
/* ... */
"contacts_options_button" = "Options";
```
3. Add extension to `String` to make it easier to use localized strings. You can take some code from Antidote [link](https://github.com/Antidote-for-Tox/Antidote/blob/master/Antidote/StringExtension.swift#L27-L30).
4. Here use following code `callButton.title = String(localized: "contacts_call_button")`